### PR TITLE
Add Lyon bicycle parking integration

### PR DESCRIPTION
**Pull request by XioNoX**
From a discussion on https://forum.openstreetmap.fr/t/osmose-et-les-parkings-a-velo/25322

### CODE REVIEW

File: analysers/analyser_merge_bicycle_parking_FR_lyon.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        self.def_class_update_official(item = 8152, id = 4, level = 3, tags = ['merge'],
+            title = T_('Bicycle parking update'))
+        self.init(
+            "https://www.data.gouv.fr/fr/datasets/parcs-de-stationnement-velos-de-la-metropole-de-lyon/",
+            "Localisation des stationnements vélos connus sur le territoire de la Métropole de Lyon",
+            GeoJSON(
+                Source(attribution = "Métropole de Lyon",
+                    fileUrl = "https://data.grandlyon.com/geoserver/metropole-de-lyon/ows?SERVICE=WFS&VERSION=2.0.0&request=GetFeature&typename=metropole-de-lyon:pvo_patrimoine_voirie.pvostationnementvelo&outputFormat=application/json&SRSNAME=EPSG:4326"),
+            ),
+            Load_XY("geom_x", "geom_y", where = lambda res: res["validite"] == "Validé"),
```
Comment (frodrigo) [inline analysers/analyser_merge_bicycle_parking_FR_lyon.py:42]: Please, could you use a `select` in place of `where` here.

https://github.com/osm-fr/osmose-backend/blob/dev/doc/4-Merge.md#data-records

File: analysers/analyser_merge_bicycle_parking_FR_lyon.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+from modules.OsmoseTranslation import T_
+from .Analyser_Merge import Analyser_Merge_Point, Source, GeoJSON, Load_XY, Conflate, Select, Mapping
+
+class Analyser_Merge_Bicycle_Parking_FR_Lyon(Analyser_Merge_Point):
+    def __init__(self, config, logger = None):
+        Analyser_Merge_Point.__init__(self, config, logger)
+        self.def_class_missing_official(item = 8150, id = 1, level = 3, tags = ['merge', 'public equipment', 'bicycle', 'fix:survey', 'fix:picture'],
+            title = T_('Bicycle parking not integrated'))
+        self.def_class_possible_merge(item = 8151, id = 3, level = 3, tags = ['merge'],
```
Comment (frodrigo) [inline analysers/analyser_merge_bicycle_parking_FR_lyon.py:31]: Use the same `tags`.

File: analysers/analyser_merge_bicycle_parking_FR_lyon.py (hunk 1 above)
Comment (XioNoX) [reply analysers/analyser_merge_bicycle_parking_FR_lyon.py]: Fixed, thanks.

File: analysers/analyser_merge_bicycle_parking_FR_lyon.py (hunk 2 above)
Comment (XioNoX) [reply analysers/analyser_merge_bicycle_parking_FR_lyon.py]: Updated as well.

**Comment by frodrigo** [comment]
Thank you. Merged, but not deployed, yet.

**Pull request merged**